README.md

```diff
@@ -32,7 +32,15 @@ Where `play-xx` is `play-25`, `play-26` or `play-27` depending on your version o
 
 ## Usage
 
-Examples can be found [here](https://github.com/hmrc/http-verbs/blob/master/http-verbs-common/src/test/scala/uk/gov/hmrc/examples/Examples.scala)
+Examples can be found [here](https://github.com/hmrc/http-verbs/blob/master/http-verbs-common/src/test/scala/uk/gov/hmrc/http/examples/Examples.scala)
+
+### URLs
+
+Urls can be supplied as either java.net.URL objects or as strings (java.net.URL is recommended). A URL interpolator has been implicitly provided with http-verbs for convenience. 
```

Comment:
Can we expand: "We recommend that you use the url interpolator for correct escaping of query and path parameters"